src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/IActiveConfiguredProjectsLoadedDataSource.cs

```diff
@@ -0,0 +1,8 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio.ProjectSystem;
+
+[ProjectSystemContract(ProjectSystemContractScope.UnconfiguredProject, ProjectSystemContractProvider.Private, Cardinality = Composition.ImportCardinality.ExactlyOne)]
+internal interface IActiveConfiguredProjectsLoadedDataSource : IProjectValueDataSource<IEnumerable<ConfiguredProject>>
+{
+}
```

Comment:
Please add a summary for this description that explains what it publishes and why you might want to use it.

Reply:
I'll this in https://github.com/dotnet/project-system/pull/8543 as a cleanup commit